README.md

```diff
@@ -6,16 +6,16 @@
 
 Gen: A General-Purpose Probabilistic Programming System with Programmable Inference
 
-*Warning: This is rapidly evolving research software.*
+*Warning: This is a rapidly evolving research software.*
```

Comment:
Since "software" is an uncountable noun, it cannot be referred to with the article "a".  One might say
> *Warning: This is a rapidly evolving research software project.*

Even better would be to remove the so-called "unqualified" _this_, for example:
> *Warning: This repository contains rapidly evolving research software.*

Reply:
Oh right, this flew under my radar. Now that you have noted this. I'm fine with the original text, as well as any of your suggestions.